utils/zerovec/src/varzerovec/slice.rs

```diff
@@ -94,6 +94,16 @@ impl<T: VarULE + ?Sized> VarZeroSlice<T> {
         }
     }
 
+    /// Uses a `&[u8]` buffer as a `VarZeroSlice<T>` without any verification.
+    ///
+    /// # Safety
+    ///
+    /// `bytes` need to be an output from [`VarZeroSlice::as_bytes()`].
+    pub const unsafe fn from_bytes_unchecked(bytes: &[u8]) -> &Self {
```

Comment:
I'm not actually using this function, but it shows up in the databake impl for VarZeroSlice, so we should probably add it.